react/tic-tac-toe/src/redux/data/actions.js

```diff
@@ -0,0 +1,21 @@
+import dataFetch from '../../services/MatchesService';
+
+export const actions = {
+  GET_DATA_REQUEST: '@@DATA/GET_DATA_REQUEST',
+  GET_DATA_SUCCESS: '@@DATA/GET_DATA_SUCCESS',
+  GET_DATA_FAILURE: '@@DATA/GET_DATA_FAILURE'
+};
+
+const actionsCreators = {
+  getData: () => dispatch => {
+    dispatch({ type: actions.GET_DATA_REQUEST });
```

Comment:
What does this action do?

Reply:
i forgot change the value of state: loading... GET_DATA_REQUEST is for that.!!! 😳😳😳